lib/trento/application/usecases/clusters/adapters/gen.ex

```diff
@@ -0,0 +1,7 @@
+defmodule Trento.Clusters.Gen do
```

Comment:
I was thinking that this abstraction name `Trento.Clusters` might be narrowed down to what could be thought as the `ChecksEngine`, or similar names, as its responsibility is to provide an entry point interface to functionalities about executing checks, not really providing abstraction to the whole cluster thing.

It might become something like

```elixir
# config/config.exs
config :trento, Trento.Clusters, checks_engine: Trento.Checks.Engine.Runner

# config/wanda.exs
config :trento, Trento.Clusters, checks_engine: Trento.Checks.Engine.Wanda

# lib/trento/application/usecases/clusters/clusters.ex
defmodule Trento.Clusters do
  ...
  @spec request_checks_execution(String.t()) :: :ok | {:error, any}
  def request_checks_execution(cluster_id) do
    engine().request_checks_execution(cluster_id)
  end
  ...
  defp engine do
    Application.fetch_env!(:trento, __MODULE__)[:checks_engine]
  end
end
```

Reply:
Ok!
This adapter is a temporary thing, but well, let's name it as it is